src/AutoRest.CSharp/LowLevel/AutoRest/DpgOutputLibrary.cs

```diff
@@ -58,7 +58,7 @@ public DpgOutputLibrary(string libraryName, IReadOnlyDictionary<InputEnumType, E
 
         private IEnumerable<string>? _accessOverriddenModels;
         public IEnumerable<string> AccessOverriddenModels => _accessOverriddenModels ??= Enums.Where(e => e.IsAccessibilityOverridden).Select(e => e.Declaration.Name)
-            .Concat(Models.Where(m => m.IsAccessibilityOverridden).Select(m => m.Declaration.Name));
+            .Concat(Models.Where(m => m.IsAccessibilityOverridden && (m.Namespace == null || !m.Namespace.StartsWith("Azure.Core"))).Select(m => m.Declaration.Name));
```

Comment:
Well if this is the reason, I think we do not need to fix this because this really looks like a workaround, and we should not do this.

Reply:
If we have to fix, maybe we should let emitter not generate models from azure.core. There is a flag in TCGC that to filter out azure.core models.

Reply:
we cannot let TCGC filter out core models, because that will break other cases for emitter. and I remember in generator we already filter out core models?

Reply:
our emitter could do this.

Reply:
OK. Previously I thought our generator will remove core types. If that assumption is false, then it makes sense to filter out in emitter.